knees4bees/dream-up-pup
# Feature/basic setup

### What does this PR do?
Creates basic structure for app

### Are there any known bugs?
Nope

### Summary of the changes
* Removes unneeded files that were created by Create React App
* Adds logo and icon files
* Creates skeleton structure for components

### Notes for the reviewer
n/a

### How should this be tested?
View app in browser and confirm that "hi" appears on the page with no warnings in the console.
